cardano-cli/src/Cardano/CLI/EraBased/Commands.hs

```diff
@@ -28,12 +30,14 @@ renderAnyEraCommand :: AnyEraCommand -> Text
 renderAnyEraCommand = \case
   AnyEraCommandOf _ cmd -> renderEraBasedCommand cmd
 
-newtype EraBasedCommand era
+data EraBasedCommand era
   = EraBasedGovernanceCmds (EraBasedGovernanceCmds era)
+  | TransactionCmds (TransactionCmds era)
```

Comment:
Do we also want to name this `EraBasedTransactionCmds`?

Reply:
This doesn't break existing commands.

My thoughts are we don't need to have the `EraBased` prefix because we already qualify the legacy stuff with `Legacy` stuff and when we get to the point where we can delete the legacy stuff `EraBased` is just the normal stuff.

Reply:
I plan to remove `EraBased` from prefix from the governance commands.